flake.lock

```diff
@@ -1,23 +1,29 @@
 {
   "nodes": {
-    "fenix": {
+    "crane": {
       "inputs": {
+        "flake-compat": [
+          "flake-compat"
+        ],
+        "flake-utils": [
+          "flake-utils"
+        ],
         "nixpkgs": [
           "nixpkgs"
-        ],
-        "rust-analyzer-src": "rust-analyzer-src"
+        ]
       },
       "locked": {
-        "lastModified": 1663570974,
-        "narHash": "sha256-ncUdRdY70VdJIX6Mi+820xeD7FutADd3NbQR0BKkFYA=",
-        "owner": "nix-community",
-        "repo": "fenix",
-        "rev": "02093d3aca186135da78b76ac28ec58031391076",
+        "lastModified": 1665490127,
+        "narHash": "sha256-ZQdaPEgnR8AZGSjQ5dUYz8+D+EsKG4GymmDenBBwT30=",
+        "owner": "rvolosatovs",
```

Comment:
Why use your fork rather than upstream? Can we leave a comment with a link to an issue regarding using upstream instead? And if we must use a fork in the long-term, should we fork it into the Enarx organization?

Reply:
It will go away in a few days, I just need to get back to the PR referenced in the comment in the flake

I can create an issue and reference in a follow-up PR

Reply:
https://github.com/enarx/enarx/pull/2269/files#diff-206b9ce276ab5971a2489d75eb1b12999d4bf3843b7988cbe8d687cfde61dea0R7